offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/update/UpdatePersonView.groovy

```diff
@@ -156,7 +156,7 @@ class UpdatePersonView extends CreatePersonView {
         })
 
         updatePersonViewModel.addPropertyChangeListener("affiliationValid", {
-            if (updatePersonViewModel.affiliationValid || updatePersonViewModel.affiliationValid == null) {
+            if (updatePersonViewModel.affiliationValid) {
```

Comment:
We need this. Otherwise the component error will show.

Reply:
I didn't notice any component errors after removing this check.

Reply:
The only issue i found was that the previous address additions from an added organisation were selectable.

Reply:
https://user-images.githubusercontent.com/29627977/121348532-c2435700-c928-11eb-93de-fdf9e561739a.mov

Reply:
you need to make sure the affiliation triggers a reevaluation of the valid property upon reset.

Reply:
I think the button should be bound to the affiliation (being present && valid)

Reply:
compare to setup of the create person and update person buttons